doc/doxygen/src/Configuration.dox

```diff
@@ -0,0 +1,174 @@
+/*!
+ * @page runtime_configuration Configuring OpenAssetIO
+ *
+ * This page covers how to configure OpenAssetIO for a specific
+ * end-user working environment.
+ *
+ * @section runtime_configuration_overview Overview
+ *
+ * @startuml
+ * node Host {
+ *   package OpenAssetIO {
+ *     collections ManagerPlugin
+ *   }
+ * }
+ *
+ * cloud "Asset Management System" as ams
+ *
+ * ManagerPlugin -right-> ams
+ * ams -left-> ManagerPlugin
+ * @enduml
+ *
+ * An OpenAssetIO enabled tool or application is known as a @ref host "Host"
+ * (as it "hosts" the API, tenuous - but naming things is hard). Through
+ * the use of a plugin system, OpenAssetIO allows the host to
```

Comment:
[Consider]
```suggestion
 * (as it "hosts" the API). Through
 * the use of a plugin system, OpenAssetIO allows the host to
```
I don't think this aside is that helpful for comprehension.

Reply:
But the whimsy, the whimsy! 🤣 Removed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/860/commits/da4b28697cbc15b230dc72e29c13af56e80bdb50)